vowpalwabbit/parse_args.cc

```diff
@@ -1381,7 +1392,6 @@ VW::workspace& parse_args(
 
     parse_diagnostics(*all.options, all);
 
-    all.initial_t = static_cast<float>(all.sd->t);
```

Comment:
Why this change?

Reply:
This line has been moved to `1093` in the same file. The entire option_group_definition had to be moved to a later section in the parser, namely after
`load_header_merge_options(*all.options, all, *model, interactions_settings_duplicated);`
is called so that we have access to the loaded arguments. This line needs to be moved because it is set of the parsed arguments.

Reply:
Sounds good.